packages/story-editor/src/utils/elementBorder.js

```diff
@@ -194,3 +198,28 @@ export function getResponsiveBorder(border, previewMode, converter) {
     bottom: converter(bottom),
   };
 }
+
+/**
+ * Gets style for the border svg wrapper.
+ *
+ * @param {string} maskId Clipping mask id.
+ * @param {number} borderWidth border width value
+ * @return {Object} SVG border wrapper style.
+ */
+export function getBorderWrapperStyle(maskId, borderWidth) {
+  if (!maskId || !borderWidth) {
+    return {};
+  }
```

Comment:
Could this check for the rectangular mask as well and just return nothing here?

Reply:
Yes, it could